api/setup.ts

```diff
@@ -0,0 +1,65 @@
+//Imports
+import { parse,  stringify } from "https://deno.land/std@0.97.0/encoding/yaml.ts";
+import { DOMParser, Element } from "https://deno.land/x/deno_dom/deno-dom-wasm.ts";
+
+//Load static/base.html
+let html: any = new DOMParser().parseFromString(await Deno.readTextFile("./static/base.html"), "text/html")!
```

Comment:
```suggestion
let html: any = new DOMParser().parseFromString(await Deno.readTextFile("./static/base.html"), "text/html")
```
Extra `!` ? Or does it has a special effect ?